requirements.in

```diff
@@ -20,15 +20,14 @@ python-slugify
 requests
 social-auth-app-django
 wagtail-airtable
-wagtail==2.11.7
+wagtail==2.12.4
```

Comment:
We can't upgrade to Wagtail 2.13 and Django 3.2 until wagtail-modeltranslation supports Django 3.2 and Wagtail 2.13. But we'll be removing modeltranslation in #6705